tutorials/fundamentals/then-async.md

```diff
@@ -0,0 +1,33 @@
+When making API requests, you can either use `.then()` or `async/await`. Internally, they are the same thing.
+The difference is that in an `async/await` function, the entire function execution will be halted on the first instance of a promise whereas when using
+`.then()`, only the `.then()` block execution will be halted.
```

Comment:
The `then()` block execution bit isn't clear. Maybe something more like "the rest of the function will continue to execute, but JavaScript won't execute `then()` callback until the promise settles."